tests/test_point_density.py

```diff
@@ -149,6 +160,60 @@ def test_point_density_ppf(scale: Scale):
     assert uniform_dist.ppf(1) == pytest.approx(scale.high, rel=0.1)
 
 
+@pytest.mark.parametrize(
+    "scale", [Scale(-3, 10), LogScale(0.01, 5, 500)],
+)
+def test_point_density_modes(scale: Scale):
+    dist = get_dist_from_scale(scale)
+
+    modes = dist.modes()
+
+    assert len(modes) > 0
+
+    mode_densities = np.array([dist.pdf(mode) for mode in modes])
+
+    assert np.all(mode_densities == mode_densities[0])
+
+    mode_density = mode_densities[0]
+
+    # It's important to test against dist.to_arrays()
+    # instead of e.g. dist.true_xs.
+    # The modes are for the exported xs, not for the true_xs.
+    # See the .modes function docstring for more.
```

Comment:
It doesn't really seem like there's more detail in the `.modes` docstring?

Reply:
I'm just going to delete this comment, it's clearly more inaccurate and confusing than it is illuminating :p